client/src/js/services/receipts/modal/receiptModal.tmpl.html

```diff
@@ -53,6 +53,6 @@
 </div>
 
 <div class="modal-footer">
-  <button class="btn btn-default btn-sm" ng-click="ReceiptCtrl.close()">Close</button>
-  <button class="btn btn-primary btn-sm" ng-click="ReceiptCtrl.print()"><span class="glyphicon glyphicon-print"></span> Print</button>
+  <button class="btn btn-default btn-sm" ng-click="ReceiptCtrl.close()" data-action="close">Close</button>
```

Comment:
These should eventually be translated. I believe this was first introduced by @sfount.